R/net.inputs.R

```diff
@@ -1033,9 +1049,13 @@ crosscheck.net <- function(x, param, init, control) {
     if (control[["start"]] == 1 && control[["skip.check"]] == FALSE) {
 
       # Main class check ----------------------------------------------------
-      if (!inherits(x, c("netest", "netsim"))) {
-        stop("x must be either an object of class netest or class netsim",
-             call. = FALSE)
+      if (inherits(x, "netest")) {
+        x <- list(x)
+      }
+      if (!inherits(x, "list") || length(x) == 0 ||
+          !all(unlist(lapply(x, inherits, "netest")))) {
```

Comment:
could we use 
`vapply(x, inherits, "netest", logical(1))` instead?

Reply:
I've switched to `vapply` and `inherits` as suggested.